amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -155,6 +155,11 @@ function exportScorecard () {
 	window.location =  "/rest/scorecard/export";
 }
 
+function viewDifferences(activityOneId) {
+    document.getElementById("method").value = "viewDifferences";
+    document.getElementById("activityOneId").value = activityOneId;

```

Comment:
now we don't have another method or activityOneId but just in case we add one in the feature, lets use document.aimCompareActivityVersionsForm instead getting the element by id. To avoid collision on another form